frame/aura/src/lib.rs

```diff
@@ -102,7 +102,7 @@ pub mod pallet {
 
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
-		fn on_initialize(_: T::BlockNumber) -> Weight {
+		fn on_initialize(_: BlockNumberFor<T>) -> Weight {
```

Comment:
This is a good example for what I suggested above, because we already have `frame_system::pallet_prelude` in scope :)